.github/workflows/pulp_images.yml

```diff
@@ -128,19 +140,15 @@ jobs:
           - image_name: pulp-minimal
             web_image: pulp-web
             pip_name: pulpcore
-            s6: false
```

Comment:
If you think this improves how easy it is to understand pulp_images.yml, I am OK with this.